src/containers/shared/components/Streams/StreamsProvider.tsx

```diff
@@ -55,7 +55,7 @@ const truncateLedgers = (ledgers: Record<string, Ledger>, count) =>
 export const StreamsProvider: FC = ({ children }) => {
   // In custom mode we populate metrics from ledgers loaded into memory
   const useServerMetrics = process.env.VITE_ENVIRONMENT !== 'custom'
-  const [ledgers, setLedgers] = useState<Record<number, Ledger>>([])
+  const [ledgers, setLedgers] = useState<Record<number, Ledger>>({})
```

Comment:
hmm, nice catch! I wonder why the tsc did not complain about the type mismatch before. There must be some form of implicit type coercion in the compiler.